content/c-sharp/concepts/math-functions/terms/min/min.md

```diff
@@ -0,0 +1,57 @@
+---
+Title: '.Min()'
+Description: 'Returns the smaller of two specified numbers.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Methods'
+  - 'Numbers'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+# .Min()
```

Comment:
Remove this, we don't place h1 headers in entries, the metadata title will do this for us.

Reply:
header removed